stable/distribution/templates/database-secrets.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
@eldada Can you add condition check here. Create secret only if values are provided in .`Values.database` and postgress and mondo are disabled.